tuning/tune.py

```diff
@@ -839,17 +881,19 @@ def generate_constraints(
     constraints += [wg_x == subgroup_size * subgroup_n_count]
     constraints += [wg_y == subgroup_m_count]
     constraints += [wg_z == subgroup_k_count]
-    constraints += [wg_x <= m, wg_x <= n]
+    constraints += [z3.Or(wg_x <= n, wg_x <= m)]
```

Comment:
does it help on producing more valid candidates?

Reply:
yes